### PR TITLE
techjunkie.com, manuall.*, handleidingkwijt.com, shuomingshu88.com

### PR DESCRIPTION
**Pull request by DandelionSprout**
Example pages:

```
! https://www.techjunkie.com/
techjunkie.com##.qualcont
! https://manuall.co.uk/asus-xg-u2008-switch/
! https://handleidingkwijt.com/asus-xg-u2008-switch/
! https://shuomingshu88.com/leicatrinovid-10x42shuangtongwangyuanjing/
manuall.*,handleidingkwijt.com,shuomingshu88.com##.col-sm-9 > div:has-text(advertisement)
```

### CODE REVIEW

File: dist/placeholder-buster.txt [hunk 1]
```diff
@@ -129,6 +129,9 @@ rpmseek.com###lbanner_top
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/41#issuecomment-515740473
 edition.cnn.com###story-bottom
 
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/44
+manuall.*,handleidingkwijt.com,shuomingshu88.com##.col-sm-9 > div:has-text(advertisement)
```
Comment (jspenguin2017) [inline dist/placeholder-buster.txt:133]: Can you provide test links for the other domains as well?

**Review by jspenguin2017 — CHANGES_REQUESTED** [review]

File: dist/placeholder-buster.txt (hunk 1 above)
Comment (DandelionSprout) [reply dist/placeholder-buster.txt:133]: They're pretty much all the same site, just that two of the regional versions had to use different domain names. But nevertheless I've done that now.

**Review by jspenguin2017 — APPROVED** [review]

**Comment by jspenguin2017** [comment]
`https://www.techjunkie.com/`

**Comment by jspenguin2017** [comment]
`https://manuall.co.uk/asus-xg-u2008-switch/`
`https://handleidingkwijt.com/asus-xg-u2008-switch/`
`https://shuomingshu88.com/leicatrinovid-10x42shuangtongwangyuanjing/`

**Pull request merged**